apps/web/app/(all)/[workspaceSlug]/(settings)/settings/projects/[projectId]/automations/layout.tsx

```diff
@@ -6,16 +6,14 @@ import { Outlet } from "react-router";
 import { AutomationsListWrapper } from "@/plane-web/components/automations/list/wrapper";
 import type { Route } from "./+types/layout";
 
-const AutomationsListLayout = observer((props: Route.ComponentProps) => {
-  const {
-    params: { projectId, workspaceSlug },
-  } = props;
+function AutomationsListLayout({ params }: Route.ComponentProps) {
```

Comment:
@prateekshourya29 it appears this layout is not in the routes.ts file, you might of dropped a few routes during your refactor. and tbh, the indirection you added is a lot harder to read, I would revert those changes, it might be noisy, but its much easier to make sense of.